data/hai/hai reveal 4 middle.txt

```diff
@@ -736,7 +741,7 @@ mission "ZZ Hai Reveal [A04] Report to Teeneep"
 			has "Hai Reveal: Pirate Troubles [4]: declined"
 			has "hr defeated scar's legion"
 	on offer
-		dialog `Alondo is apparently away somewhere and unavailable for a short while. So, now that the situation with Scar's Legion is resolved, it's time to report to Teeneep about the experience. Maybe it will shed some light on what's going on?`
+		dialog `You have learned of information to report to Alondo, but apparently he is away somewhere and unavailable for a short while. You should also report this to Teeneep, and waiting for Alondo could be suspicious. So, now that the situation with Scar's Legion is resolved, it's time to report to Teeneep. Maybe it will shed some light on what's going on?`
```

Comment:
`You should also report this to Teeneep, and waiting for Alondo could be suspicious. So, now that the situation with Scar's Legion is resolved, it's time to report to Teeneep. Maybe it will shed some light on what's going on?` 
This sounds repetitive.

Reply:
True, thoughts on this? otherwise I might summon Ejo
```suggestion
		dialog `You have learned of information to report to Alondo, but apparently he is away somewhere and unavailable for a short while. So, now that the situation with Scar's Legion is resolved, this is also something you should also report this to Teeneep, and waiting for Alondo could be suspicious. Maybe going to Teeneep will shed some light on what's going on?`
```

Reply:
Delete the extra "this" in the suggestion and I think it sounds better.

Reply:
~~you saw nothing, it was never there, I can construct a coherent sentence~~

Reply:
Better look will have to wait until after work, since my breaks are pretty used up tonight, but I like this change as it is now :)

Reply:
It sounds better, but there's something off with it that I'm too tired to think of yet (maybe too long?). At least it's not repetitive. I will look at it in the morning and check around the context.